.changeset/ninety-lions-do.md

```diff
@@ -0,0 +1,6 @@
+---
+"@quri/squiggle-lang": patch
+"@quri/squiggle-components": patch
+---
+
+Breaking: Removes points(number) from DistFns and NumericFns, replaces with xPoints(number[])
```

Comment:
Reminder that these notes get into public changelogs; "Removed" or "Remove" would read better (also `` would be nice for formatting).

Might be useful to write a style guide for changelogs eventually...